Formula/archey4.rb

```diff
@@ -0,0 +1,32 @@
+class Archey4 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Simple system information tool written in Python"
+  homepage "https://github.com/HorlogeSkynet/archey4"
+  url "https://files.pythonhosted.org/packages/68/7d/8066c39289bda86c606350aca46b05e157eed2b6391af75b5745cbd6f61c/archey4-4.11.0.tar.gz"
+  sha256 "18113b18282ea0ab31acd74d22d2830160ddf0efc82438e328b3ca9823e15666"
+  license "GPL-3.0-or-later"
+
+  depends_on "python@3.9"
+
+  resource "distro" do
+    url "https://files.pythonhosted.org/packages/a6/a4/75064c334d8ae433445a20816b788700db1651f21bdb0af33db2aab142fe/distro-1.5.0.tar.gz"
+    sha256 "0e58756ae38fbd8fc3020d54badb8eae17c5b9dcbed388b17bb55b8a5928df92"
+  end
+
+  resource "netifaces" do
+    url "https://files.pythonhosted.org/packages/0d/18/fd6e9c71a35b67a73160ec80a49da63d1eed2d2055054cc2995714949132/netifaces-0.10.9.tar.gz"
+    sha256 "2dee9ffdd16292878336a58d04a20f0ffe95555465fee7c9bd23b3490ef2abf3"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    ENV["DO_NOT_TRACK"] = "1"
```

Comment:
Why is this needed?

Reply:
For privacy purposes, as running `archey -j` would automatically load internal `WAN_IP` entry performing outgoing requests. Setting this allows us to prevent this behavior without having to deal with configuration file.

Reply:
I don't think our CI system has any privacy requirements. I'd say this can go.

Reply:
Oh I was actually thinking about end-users/developers tweaking formulae (or just this one for instance) and running `brew test <...>` on their own system, rather than the CI IP address.
What do you think about this eventual case ?

Reply:
They probably have the tool installed and use it too. I don't think we should make the test more complicated than it needs to be.

Reply:
I don't want to bother you much but :

* Maybe they use this tool, but with this very variable exported in their shell profile :wink: 

* Sadly, I've just checked, `ENV` seems to be reset by `brew` when running tests (this means a developer exporting this variable in their profile would have no impact in this case)

* When coding Archey unit tests I particularly tried to take care of this issue, and I guess packaging/distributions scripts should try to do the same :roll_eyes:

* Final 2 cents : I don't think an environment variable complicates a test much, do it ?

Thanks for your time :wave:

Reply:
Hey @SMillerDev ! The boring guy is back. What do you think about my previous points ? Thanks, bye 👋

Reply:
I appreciate your concern here but I think the reality is that `brew test` is rarely used outside of CI, and when it is called manually it's usually done by maintainers.    

I don't have a strong feeling about this but I'm usually in favor of keeping tests as simple as possible.  There are certainly tests where we have to set some environment variables in order for them to work in CI, but this doesn't appear to be the case.  So I would say we shouldn't include it.

Reply:
Please remove this variable to move this pull request forward, thanks.